test/pkg/kernel-config/Makefile

```diff
@@ -1,7 +1,7 @@
 .PHONY: tag push
 
 BASE=alpine:3.5
-IMAGE=check
+IMAGE=kernel-config
```

Comment:
Ah! sorry. i thought this was updated but appears git crashed. Will push an update in a jiffy!